tests/system/test_network_part.py

```diff
@@ -113,17 +116,17 @@ def test_mom_with_network_failure_bounce_master():
     installAppCurlCommand = "curl -X POST -H 'Authorization: token=" + TOKEN + "' " + DCOS_SERVICE_URL + "/v2/apps -d @large-sleep.json --header 'Content-Type: application/json' "
     run_command_on_master(installAppCurlCommand)
 
-    wait_for_task("marathon-user","sleep")
+    wait_for_task("marathon-user", "sleep")
 
     # grab the sleep taskID
-    json = get_json(__marathon_url("apps/sleep","marathon-user"))
+    json = get_json(__marathon_url("apps/sleep", "marathon-user"))
     original_sleep_task_id = json["app"]["tasks"][0]["id"]
 
     mom_task_ips = get_service_ips("marathon-user", "sleep")
     for task_ip in mom_task_ips:
         break
 
-    print ("\nTask IP: " + task_ip)
+    print("\nTask IP: " + task_ip)
```

Comment:
For another pull request, but this file has a mixture of `"some string" + var` and `"some string{}".format(var)`. It would be great to pick one style and use it throughout the code base.